[2024-08-14T05:07:01.342+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-14T05:07:01.369+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Weather_Data_Extraction.generate_output_file backfill__2024-03-29T00:00:00+00:00 [queued]>
[2024-08-14T05:07:01.389+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Weather_Data_Extraction.generate_output_file backfill__2024-03-29T00:00:00+00:00 [queued]>
[2024-08-14T05:07:01.390+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-08-14T05:07:01.579+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): generate_output_file> on 2024-03-29 00:00:00+00:00
[2024-08-14T05:07:01.586+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3087) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-14T05:07:01.587+0000] {standard_task_runner.py:63} INFO - Started process 3089 to run task
[2024-08-14T05:07:01.588+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Weather_Data_Extraction', 'generate_output_file', 'backfill__2024-03-29T00:00:00+00:00', '--job-id', '668', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/Data_Weather_Extraction.py', '--cfg-path', '/tmp/tmpdwilj71m']
[2024-08-14T05:07:01.591+0000] {standard_task_runner.py:91} INFO - Job 668: Subtask generate_output_file
[2024-08-14T05:07:01.661+0000] {task_command.py:426} INFO - Running <TaskInstance: Weather_Data_Extraction.generate_output_file backfill__2024-03-29T00:00:00+00:00 [running]> on host ubuntu
[2024-08-14T05:07:01.818+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Ducky' AIRFLOW_CTX_DAG_ID='Weather_Data_Extraction' AIRFLOW_CTX_TASK_ID='generate_output_file' AIRFLOW_CTX_EXECUTION_DATE='2024-03-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2024-03-29T00:00:00+00:00'
[2024-08-14T05:07:01.822+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-14T05:07:01.862+0000] {logging_mixin.py:188} INFO - Generated output file: Data_Weather-20240329.csv
[2024-08-14T05:07:01.863+0000] {logging_mixin.py:188} INFO - Execution Date: 2024/03/29
[2024-08-14T05:07:01.899+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-14T05:07:01.900+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-14T05:07:01.914+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Weather_Data_Extraction, task_id=generate_output_file, run_id=backfill__2024-03-29T00:00:00+00:00, execution_date=20240329T000000, start_date=20240814T050701, end_date=20240814T050701
[2024-08-14T05:07:01.964+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-14T05:07:01.991+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-14T05:07:01.994+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
